tests/cnf/core/network/dpdk/tests/rootless.go

```diff
@@ -331,14 +340,19 @@ var _ = Describe("rootless", Ordered, Label(tsparams.LabelSuite), ContinueOnFail
 				Expect(err).ToNot(HaveOccurred(), "Fail to collect PCI addresses")
 
 				By("Running client dpdk-testpmd")
-				rxTrafficOnClientPod(clientPod, defineTestPmdCmd(tapOneInterfaceName, pciAddressList[0]))
+				err = cmd.RxTrafficOnClientPod(clientPod, defineTestPmdCmd(tapOneInterfaceName, pciAddressList[0]))
+				Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(
+					"The Receive traffic test on the the client pod failed %s", err))
 
 				By("Checking the rx output of tap ext0 device")
 				checkRxOutputRateForInterfaces(
 					clientPod, map[string]int{
 						tapOneInterfaceName:         minimumExpectedDPDKRate,
 						firstInterfaceBasedOnTapOne: minimumExpectedDPDKRate})
-				rxTrafficOnClientPod(clientPod, defineTestPmdCmd(tapTwoInterfaceName, pciAddressList[1]))
+				err = cmd.RxTrafficOnClientPod(clientPod, defineTestPmdCmd(tapTwoInterfaceName, pciAddressList[1]))
+				Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(
```

Comment:
Remove fmt.Sprintf, the error will shown anyway

Reply:
removed